301-drupal8-vmss-glusterfs-mysql/azuredeploy.json

```diff
@@ -99,22 +99,23 @@
       "type": "string",
       "defaultValue": "no",
       "allowedValues": [
-        "no"
+        "no",
+        "yes"
       ],
       "metadata": {
         "description": "If Yes new MySQL server will be provisioned using the mysql replication template"
       }
     },
     
-    "mySqlFQDN": {
+    "existingMySqlFQDN": {
```

Comment:
For existing parameters, please include default value = "" otherwise user needs to specify something in Portal